### PR TITLE
Update `Setting Up Volumes.md`

### PR DESCRIPTION
**Pull request by outdooracorn**
- **add links to the "Volume Types" list** 
  - Add links from the "Volume Types" list at the top of the page to their "Creating <volume_type> Volumes" section further down
- **fix order of "Volume Types" list** 
  - The "Creating <volume_type> Volumes" sections are in a different order to the list of "Volume Types" at the top of the page. Deprecated volumes (e.g. Striped) are shown before non-deprecated volumes (e.g. Distributed Replicated).
- **removed 'Striped' volumes**
  - Removing reference to the 'Striped' volumes in `Setting Up Volumes.md` as they don't work in the latest versions (glusterfs-6.x and glusterfs-7.x)

### CODE REVIEW

File: docs/Administrator Guide/Setting Up Volumes.md [hunk 1]
```diff
@@ -581,6 +409,172 @@ volumes, but using dispersed subvolumes instead of replicated ones.
 
     > Use the `force` option at the end of command if you want to create the volume in this case.
 
+## Creating Striped Volumes
```
Comment (amarts) [inline docs/Administrator Guide/Setting Up Volumes.md]: do we need this as with latest version (glusterfs-6.x and glusterfs-7.x), these commands doesn't even work. I am OK with removing them altogether.

Comment (outdooracorn) [reply docs/Administrator Guide/Setting Up Volumes.md]: Oh, I didn't realise they no longer worked, only just started using GlusterFS. Will push another commit to remove them. Is it all the deprecated volumes that have been removed or just the Striped ones?

Comment (amarts) [reply docs/Administrator Guide/Setting Up Volumes.md]: > only just started using GlusterFS.

Welcome! and much appreciated contribution Indeed.

We can remove the 'Striped' ones in this PR, and pick others in follow up patches as and when you encounter differences.

Comment (outdooracorn) [reply docs/Administrator Guide/Setting Up Volumes.md]: > Welcome! and much appreciated contribution Indeed.

Thanks for the kind words and for your contributions as well! 😁

> We can remove the 'Striped' ones in this PR, and pick others in follow up patches as and when you encounter differences.

Realised when I looked again all the deprecated ones are 'Striped'! I've pushed another commit removing all the deprecated 'Striped' sections.

Comment (amarts) [reply docs/Administrator Guide/Setting Up Volumes.md]: We normally 'squash and merge' patches. So no need for that.

**Pull request merged**